main.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'app'
+require_relative 'files_manager'
+require_relative 'game'
+require_relative 'book'
+
+@file_manager = FileManager.new
+GameTemplate = Struct.new(:genre, :author, :source, :label, :publish_date)
+game_in = GameTemplate.new('', '', '', '', '2012-02-02')
+@game = Game.new(game_in, true, '2022-2-2')
+
+def display_menu
+  puts 'What would you like to do?'
+  puts '1. List all books'
+  puts '2. List all genres'
+  puts '3. List all labels'
+  puts '4. List all authors'
+  puts '5. List all games'
+  puts '6. Add a book'
+  puts '7. Add a music album'
+  puts '8. Add a game'
+  puts '9. Quit'
+end
```

Comment:
- [ ] Please consider adding an option to list all music albums as required! 🤓 🔧